apps/web/src/pages/Applications/ApplicationQuestionsPage/ApplicationQuestionsPage.tsx

```diff
@@ -66,9 +76,166 @@ export const getPageInfo: GetApplicationPageInfo = ({
+          })}
+        </Link>
+      </div>
+      <p data-h2-margin="base(x1, 0)">
+        {intl.formatMessage({
+          defaultMessage:
+            'Don\'t forget to take a break if you need to! Using the "Save and quit for now" button, you can record your progress and return to questions you might be stuck on later.',
+          id: "tMnjyJ",
+          description:
+            "Notice that application can be saved and returned to at a later time.",
+        })}
+      </p>
+      <BasicForm
+        onSubmit={handleSubmit}
+        options={{
+          defaultValues: dataToFormValues(
+            screeningQuestions,
+            screeningQuestionResponses,
+          ),
+        }}
+      >
+        {screeningQuestions.length ? (
+          screeningQuestions.map((question, index) => (
+            <React.Fragment key={question.id}>
+              <Heading level="h3" size="h4">
+                {intl.formatMessage(
+                  {
+                    defaultMessage: "Question {number}",
+                    id: "/sBGov",
+                    description: "Heading for a specific screening question",
+                  },
+                  { number: index + 1 },
+                )}
+              </Heading>
+              <input type="hidden" name={`answers.${index}.id`} />
+              <input type="hidden" name={`answers.${index}.questionId`} />
+              <TextArea
+                id={`answers.${index}.answer`}
+                name={`answers.${index}.answer`}
+                label={getLocalizedName(question.question, intl)}
+                rules={{ required: intl.formatMessage(errorMessages.required) }}
+              />
+            </React.Fragment>
+          ))
+        ) : (
+          <Well>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This process has no screening questions. You may continue onto the next step.",
+                id: "jIhIaX",
+                description:
+                  "MEssage displayed to users when there are no screening questions for a proccess",
+              })}
+            </p>
+          </Well>
+        )}
+        <Separator
+          orientation="horizontal"
+          decorative
+          data-h2-background="base(black.light)"
+          data-h2-margin="base(x2, 0)"
+        />
+        <div
+          data-h2-display="base(flex)"
+          data-h2-gap="base(x.25, x.5)"
+          data-h2-flex-wrap="base(wrap)"
+          data-h2-flex-direction="base(column) l-tablet(row)"
+          data-h2-align-items="base(flex-start) l-tablet(center)"
+        >
+          <Button type="submit" mode="solid">
+            {intl.formatMessage({
+              defaultMessage: "Let's review my application",
+              id: "uPnk4X",
+              description:
+                "Button text to submit screening questions and continue to application review",
+            })}
+          </Button>
+          <Link
```

Comment:
This "save" button doesn't actually save, right?  It just navigates away?

Reply:
Looks like I missed this AC. These form actions are very confusing for me. I never know what ones actually submit the form and what ones just say "Save" to reassure users 😞 

Updated in [update save and quit to submit form](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6311/commits/624291b8843ce1e4386e8c2d2ec500d1b1c10804)